sdks/go/pkg/beam/core/runtime/graphx/serialize.go

```diff
@@ -122,15 +122,15 @@ func DecodeMultiEdge(edge *v1.MultiEdge) (graph.Opcode, *graph.Fn, *window.Fn, [
 func encodeCustomCoder(c *coder.CustomCoder) (*v1.CustomCoder, error) {
 	t, err := encodeType(c.Type)
 	if err != nil {
-		return nil, fmt.Errorf("bad underlying type: %v", err)
+		return nil, fmt.Errorf("failed to encode custom coder %v, bad underlying type: %v", *c, err)
```

Comment:
I think "underlying type" might end up being misleading, but I can't work out the phraseing.
"failed to encode custom coder %v for type %v: %v", *c,  c.Type,

Reply:
Yeah, it's a bit tricky. Your suggestion is better than anything I thought of, so I went with it.